app/vecsim_app/query.py

```diff
@@ -1,12 +1,38 @@
 import typing as t
+
 from redis.asyncio import Redis
-from redis.commands.search.field import VectorField, TagField
 from redis.commands.search.query import Query
+from redis.commands.search.indexDefinition import (
+    IndexDefinition,
+    IndexType
+)
+from redis.commands.search.field import (
+    VectorField,
+    TagField
+)
+
+
+INDEX_NAME = "products"
```

Comment:
Not a huge fan of a propagated global here. Just asking for side effects.

A couple other options
- Keep it a global but put it in ``config.py``
- Make it something more future-proof like an enum
- Class in ``index.py`` (new file) that holds index information (could also set configs)



this may be overkill but we want to show good practice here.

Reply:
Good call